packages/nextjs/app/builders/0x2b2542b313385AF7453623765383CE3cc5D3c9cF/page.tsx

```diff
@@ -0,0 +1,43 @@
+"use client";
```

Comment:
We don't need "use client", checkout https://github.com/BuidlGuidl/batch5.buidlguidl.com/pull/15#discussion_r1597931094